src/lib/denoify.ts

```diff
@@ -30,53 +31,21 @@ export async function denoify(params: {
 
     const packageJsonParsed = JSON.parse(fs.readFileSync("package.json").toString("utf8"));
 
-    const denoifyParamsFromPackageJson: {
-        replacer?: string;
-        ports?: { [portName: string]: string };
-        includes?: (string | { src: string; destDir?: string; destBasename?: string })[];
-        out?: string;
-        index?: string;
-    } = packageJsonParsed["denoify"] ?? {};
+    const denoifyParams = await getDenoifyParamsWithCosmiconfig();
 
-    {
-        const { includes, ports, replacer } = denoifyParamsFromPackageJson;
-
-        if (
-            (includes !== undefined &&
-                !includes.every(
-                    pathOrObj =>
-                        typeof pathOrObj === "string" ||
-                        (pathOrObj instanceof Object &&
-                            typeof pathOrObj.src === "string" &&
-                            (pathOrObj.destDir === undefined || typeof pathOrObj.destDir === "string") &&
-                            (pathOrObj.destBasename === undefined || typeof pathOrObj.destBasename === "string"))
-                )) ||
-            (ports !== undefined && !(ports instanceof Object && Object.keys(ports).every(key => typeof ports![key] === "string"))) ||
-            (replacer !== undefined && typeof replacer !== "string")
-        ) {
-            console.log(
-                [
-                    "Denoify configuration Error",
-                    'The "denoify" in the package.json is malformed',
-                    "See: https://github.com/garronej/my_dummy_npm_and_deno_module"
-                ].join("\n")
-            );
+    const tsconfigOutDir = getTsConfigOutDir();
 
-            process.exit(-1);
+    const denoDistPath = (() => {
+        if (params.denoDistPath !== undefined) {
+            return params.denoDistPath;
+        } else if (denoifyParams?.out !== undefined) {
+            return denoifyParams?.out;
+        } else if (tsconfigOutDir !== undefined) {
+            return path.join(path.dirname(tsconfigOutDir), `deno_${path.basename(tsconfigOutDir)}`);
+        } else {
+            throw new Error(`You should specify output directory by --out option or specify "outDir" in tsconfig.json`);
         }
```

Comment:
```suggestion
        if (params.denoDistPath !== undefined) {
            return params.denoDistPath;
        } 
        if (denoifyParams?.out !== undefined) {
            return denoifyParams?.out;
        }
        if (tsconfigOutDir !== undefined) {
            return path.join(path.dirname(tsconfigOutDir), `deno_${path.basename(tsconfigOutDir)}`);
        }
        throw new Error(`You should specify output directory by --out option or specify "outDir" in tsconfig.json`);
```
https://www.youtube.com/watch?v=EumXak7TyQ0